### PR TITLE
add translate for methods of Date

### PR DESCRIPTION
**Pull request by shuizhongyueming**
Copy the description of the methods of Date from the English version and translate it to a Chinese version。Most of the translation of the methods is copied from its related detail page, but also add some additional content from the English version

### CODE REVIEW

File: files/zh-cn/web/javascript/reference/global_objects/date/index.html [hunk 1]
```diff
@@ -131,8 +131,98 @@ <h3 id="实例属性">实例属性</h3>
...
+- {{jsxref("Date.prototype.toLocaleTimeString()")}}
+  - : 返回一个表述指定 {{jsxref("Date")}} 对象时间部分的的字符串。该字符串格式因不同语言而不同。
+- {{jsxref("Date.prototype.toString()")}}
+  - : 返回一个字符串，表示该 {{jsxref("Date")}} 对象。覆盖了 {{jsxref("Object.prototype.toString()")}} 方法。
+- {{jsxref("Date.prototype.toTimeString()")}}
+  - : 以人类易读形式返回一个 {{jsxref("Date")}} 对象时间部分的字符串，该字符串以美式英语格式化。
+- {{jsxref("Date.prototype.toUTCString()")}}
+  - : 使用UTC时区，把一个 {{jsxref("Date")}} 对象转换为一个字符串。
+- {{jsxref("Date.prototype.valueOf()")}}
+  - : 返回一个 {{jsxref("Date")}} 对象的原始值。覆盖了 {{jsxref("Object.prototype.valueOf()")}} 方法。
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/global_objects/date/index.html:225]: You have used markdown syntax in this part(in html file), which will not be rendered by [mdn/yari](https://github.com/mdn/yari). Please use definition table as follow:
https://github.com/mdn/translated-content/blob/77f8f285bb0c3352feca5b2a7f8331b1906ef0d5/files/zh-cn/web/javascript/reference/global_objects/date/index.html#L109-L120

Or you can convert this file to markdown like pr #5100

**Review by yin1999 — CHANGES_REQUESTED** [review]

File: files/zh-cn/web/javascript/reference/global_objects/date/index.html (hunk 1 above)
Comment (shuizhongyueming) [reply files/zh-cn/web/javascript/reference/global_objects/date/index.html]: @yin1999 I have updated the syntax

File: files/zh-cn/web/javascript/reference/global_objects/date/index.html [hunk 2]
```diff
@@ -131,8 +131,100 @@ <h3 id="实例属性">实例属性</h3>
 </dl>
 
 <h3 id="实例方法">实例方法</h3>
-
-<p>{{ page("/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Date/prototype", "Methods") }}</p>
+<dl>
+  <dt>{{jsxref("Date.prototype.getDate()")}}</dt>
+  <dd>根据本地时间，返回一个指定的 {{jsxref("Date")}} 对象为一个月中的哪一日（`1`-`31`）。</dd>
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/global_objects/date/index.html:136]: Please convert `inline code block` in to html syntax:

``` `1`-`31` ``` --> `<code>1</code>-<code>31</code>`

The same in the following lines ( L138 to L170 )

Comment (shuizhongyueming) [reply files/zh-cn/web/javascript/reference/global_objects/date/index.html]: @yin1999 converted

**Pull request merged**